lib/exception_hunter/error_creator.rb

```diff
@@ -61,6 +62,21 @@ def notify(error)
           ExceptionHunter::SendNotificationJob.perform_later(serialized_slack_notifier)
         end
       end
+
+      def hide_sensitive_values(error_attrs)
```

Comment:
How about using the `params_filter` from `ExceptionHunter::Middleware::RequestHunter`. Here's my thought, we could create a `DataRedacter#redact` and encapsulate what we use to whitelist sensitive data. We can then use ActiveSupport stuff